edge-hub/src/Microsoft.Azure.Devices.Routing.Core/endpoints/AsyncEndpointExecutor.cs

```diff
@@ -78,7 +78,7 @@ public AsyncEndpointExecutor(Endpoint endpoint, ICheckpointer checkpointer, Endp
 
         public EndpointExecutorStatus Status => this.machine.Status;
 
-        public virtual Task Invoke(IMessage message) => this.SendToTplHead(message);
+        public virtual Task Invoke(IMessage message, uint _, uint __) => this.SendToTplHead(message);
```

Comment:
why not provide name to these 2 arguments? will these be used in next PRs?

Reply:
these won't be used ever, this is in the AsyncEndpointExecutor implementation, not the StoringAsyncEndpointExecutor.

---
In reply to: [377458214](https://github.com/Azure/iotedge/pull/2469#discussion_r377458214) [](ancestors = 377458214)

Reply:
Would you tell me in what scenario, these 2 field will be used? If no use case, why do we need these 2 arguments?